Tribler/Core/Modules/gigachannel_manager.py

```diff
@@ -173,9 +174,17 @@ def on_channel_download_finished(dl):
         def _on_failure(failure):
             self._logger.error("Error when processing channel dir download: %s", failure)
 
+        def _on_success(_):
+            with db_session:
+                channel_upd = self.session.lm.mds.ChannelMetadata.get(public_key=channel.public_key, id_=channel.id_)
+                channel_upd_dict = channel_upd.to_simple_dict()
+            self.session.notifier.notify(NTFY_NODE, NTFY_UPDATE,
```

Comment:
The `NTFY_NODE` name is ambiguous. In this case, it refers to metadata nodes but I was initially confused and thought as a node as a network peer. Can you rename this variable?

Reply:
The `node` notation is already used in GigaChannels. I agree that this is confusing. I'll think of a better name.